model/src/w3gridmd.F90

```diff
@@ -924,13 +924,13 @@ MODULE W3GRIDMD
            LOGICAL :: SETUP_APPLY_WLV = .FALSE.
            INTEGER :: JGS_MAXITER=100
            INTEGER :: nbSel
-           INTEGER :: UNSTSCHEMES(4)
+           INTEGER :: UNSTSCHEMES(6)
            INTEGER :: UNSTSCHEME
            INTEGER :: JGS_NLEVEL = 0
            REAL*8  :: JGS_PMIN = 0.
            REAL*8  :: JGS_DIFF_THR = 1.E-10
-           REAL*8  :: JGS_NORM_THR = 1.E-20
-           REAL*8  :: SOLVERTHR_SETUP = 1.E-20
+           REAL*8  :: JGS_NORM_THR = 1.E-12
```

Comment:
Could the simple change in these values here be the cause of differences in the unstructured cases? I'm running some tests to see, but thought I'd also ask here.

Reply:
Agreed, I tested and this had no impact.